hadoop-hdfs-project/hadoop-hdfs-client/src/main/java/org/apache/hadoop/hdfs/ClientContext.java

```diff
@@ -227,8 +237,8 @@ public String getConfString() {
     return confString;
   }
 
-  public ShortCircuitCache getShortCircuitCache() {
-    return shortCircuitCache;
+  public ShortCircuitCache getShortCircuitCache(long idx) {
```

Comment:
This is fine. But what I realized is Hadoop applications don't really respect the annotation @InterfaceAudience.Private. You really want to gracefully deprecate a public method to avoid runtime issues. That is, keep the original getShortCircuitCache() method and let it call getShortCircuitCache(0);

Reply:
> That is, keep the original getShortCircuitCache() method and let it call
Sure, done